pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -134,6 +143,7 @@ type SubnetParam struct {
 
 type SubnetFilter struct {
 	Name            string `json:"name,omitempty"`
+	Description     string `json:"description,omitempty"`
```

Comment:
what's this for??

Reply:
This is coming from a gophercloud update